worker/handlers/repositroy.js

```diff
@@ -0,0 +1,67 @@
+const Joi = require('joi');
+
+const { searchRepositories } = require('../../services/github');
+const repositoryModel = require('../../db/models/repository');
+const userModel = require('../../db/models/user');
+const logger = require('../../logger');
+
+const searchRepositoriesResponseSchema = Joi.array().has(Joi.object({
+  node: Joi.object().keys({
+    owner: Joi.object(),
+    name: Joi.string(),
+    description: Joi.string().allow(''),
+    homepageUrl: Joi.string().allow(''),
+    stargazerCount: Joi.number(),
+    languages: Joi.object(),
+    createdAt: Joi.string(),
+  }),
+}));
+
+const onRepository = async (message) => {
+  // env = org/repo_name
+  // 1 repository
+  const { search: { edges } } = await searchRepositories({ queryString: message, first: 1 });
+
+  Joi.assert(edges, searchRepositoriesResponseSchema);
+
+  const repository = edges[0].node;
+  const { owner } = repository;
+
+  logger.info(JSON.stringify(edges[0].node, null, '-'));
+
+  const user = {
+    login: owner.login,
+    avatar_url: owner.avatarUrl,
+    html_url: owner.url,
+  };
+
+  logger.info(JSON.stringify(user));
+
+  Joi.assert(user, userModel.schema);
+
+  let insertedUser = await userModel.read(user);
+
+  if (!insertedUser.length) {
+    insertedUser = await userModel.insert(user);
+  }
+
+  logger.info(JSON.stringify(repository));
+
+  const formatedRepository = {
+    owner: insertedUser[0].id,
+    full_name: repository.name,
+    stargazers_count: repository.stargazerCount,
+    html_url: repository.homepageUrl ? repository.homepageUrl : '',
+    description: repository.description ? repository.description : '',
+    language: repository.languages.edges[0].name,
+  };
+
+  Joi.assert(formatedRepository, repositoryModel.schema);
+
+  const insertedRepos = await repositoryModel.insert(formatedRepository);
+  return (insertedRepos);
```

Comment:
It's enough to just return the insert promise.
Why is this `(insertedRepos)` in parentheses?